frontend/test/__support__/e2e/helpers/e2e-qa-databases-helpers.js

```diff
@@ -93,3 +94,19 @@ function recursiveCheck(id, i = 0) {
     }
   });
 }
+
+export function queryQADB(query, type = "postgres") {
```

Comment:
Nit: I'm breaking my fingers when typing this (even with the auto-complete). WDYT about `queryExternalDatabase` or `queryDataSource` or anything along those lines?

We might end up querying sources other than the QA databases, after all.


p.s. please don't let this block or postpone this PR. Just think about it and let's deal with it later.